packages/client/hmi-client/src/utils/petri-net-validator.ts

```diff
@@ -82,11 +77,9 @@ export const petrinetValidator = (petrinet: Petrinet): boolean => {
 		// console.log(potentialConnections);
 
 		// If the potential connections from the last iteration are the exact same then there is more than one petrinet body
-		if (statesToMerge.length === potentialConnections.length) {
-			console.log('#5');
-			return false;
-		}
+		if (statesToMerge.length === potentialConnections.length)
+			return 'Invalid petri net: #3. There are multiple petri net bodies';
 	} while (potentialConnections.length > 0);
 
-	return true; // All checks have been successfully passed
+	return 'Valid petri net'; // All checks have been successfully passed
```

Comment:
I think it'll be a good idea to more easily programmatically distinguish successful output from a failure. As it stands currently, the place where we call this validator function would need to know about this magic `'Valid petri net'` string and get the capitalization and whitespace exactly correct to be able to check if the result was successful.

Some options:
- Return an array of errors. If the array is empty, there were no errors so it passed. This might be overkill and confusing to the developer since we're exiting early after one error.
- Return `true` if the petri net is valid. You could make the return type `string | true` and leave a comment to explain that string is for errors. This feels messy to me but would work fine.
- Return an empty string `''` if the petri net is valid. This is also a magic string but would be easier to check for in the place that calls it.

I'm leaning toward the middle option but go with whatever you think is best.

Either way, adding a brief comment to the function header to explain the significance of each return value type is a good idea.